addon/serializers/osf-serializer.js

```diff
@@ -141,5 +141,13 @@ export default DS.JSONAPISerializer.extend({
             documentHash.meta.total_pages = Math.ceil(documentHash.meta.total / documentHash.meta.per_page);
         }
         return documentHash;
+    },
+
+    normalizeSingleResponse(_, __, ___, ____, requestType) {
+        const documentHash = this._super(...arguments);
+        if (requestType === 'findRecord' && documentHash.meta) {
+            documentHash.data.attributes.apiMeta = documentHash.meta;
```

Comment:
can we do `documentHash.data.attributes.meta = documentHash.meta` without adding `apiMeta: DS.attr()` to the model? In fact, I'd expect every resource to have a `meta` property and am surprised that there isn't.